lib/protocol/bep.proto

```diff
@@ -100,6 +100,7 @@ message FileInfo {
     uint32       permissions    = 4;
     int64        modified_s     = 5;
     int32        modified_ns    = 11;
+    string       modified_by    = 13;
```

Comment:
Yeah I was thinking about this too, and was first thinking full ID, but short ID is enough and has advantages.

(We try to guarantee that short IDs are unique in a cluster, but there are some corner cases where this may not be the case. In those cases our versioning is also broken however, so we should not worry about that for this specific case...)